cmd/spire-server/cli/command/run.go

```diff
@@ -19,29 +19,30 @@ import (
 )
 
 const (
-	defaultConfigPath      = "conf/server/server.conf"
-	defaultBindAddress     = "127.0.0.1"
-	defaultBindPort        = "8081"
-	defaultBindHTTPPort    = "8080"
-	defaultLogLevel        = "INFO"
-	defaultPluginDir       = "conf/server/plugin"
-	defaultBaseSpiffeIDTTL = 999999
-	defaultUmask           = 0077
+	defaultConfigPath    = "conf/server/server.conf"
+	defaultBindAddress   = "127.0.0.1"
+	defaultBindPort      = "8081"
+	defaultBindHTTPPort  = "8080"
+	defaultLogLevel      = "INFO"
+	defaultPluginDir     = "conf/server/plugin"
+	defaultBaseSVIDTtl   = 999999
+	defaultServerSVIDTtl = 999999
+	defaultUmask         = 0077
 )
 
 // CmdConfig represents available configurables for file and CLI options
 type CmdConfig struct {
-	BindAddress     string
-	BindPort        int
-	BindHTTPPort    int
-	TrustDomain     string
-	PluginDir       string
-	LogFile         string
-	LogLevel        string
-	BaseSpiffeIDTTL int
-
-	ConfigPath string
-	Umask      string
+	BindAddress   string
+	BindPort      int
+	BindHTTPPort  int
+	TrustDomain   string
+	PluginDir     string
+	LogFile       string
+	LogLevel      string
+	BaseSVIDTtl   int
+	ServerSVIDTtl int
```

Comment:
Maybe we can use BaseSVIDTtl as the value for Server SVID ttl transparently? Would cut down on required config, and seems to make sense because server svid is basically a base svid

Reply:
Another thought I had is that it's becoming apparent that Base SVID is probably bad terminology, and that Agent SVID feels more apt... just throwing it out there, I realize the Base SVID term is quite baked in, so I don't expect you to change it here... just something to think about :)

Reply:
Before this PR we used BaseSpiffeIDTTL for both. It was hard to test short life certs because one of them rotate while the other doesn't. I can remove the ServerSVIDTtl var because I agree with your point but we will need to address what I mentioned with at least a temporal fix.